### PR TITLE
FOFL-8986: Filter node change in dock menu 

### PR DESCRIPTION
**Pull request by mshariq-nerd**
1. Removed node connection socket in the dock menu node 
2. Show (+) icon on the filter node when added in the connection 
3. When one add filter placeholder node dropped on the canvas, user is not allowed to drop any other. 
4. Filter Label size increased in the dock menu.

### CODE REVIEW

**Comment by praveen-garg** [comment]
>When one add filter placeholder node dropped on the canvas, user is not allowed to drop any other.

We should disable it. May show some toast message why... if cannot

File: src/app/components/common/node-editor/custom-node/custom-node.component.ts [hunk 1]
```diff
@@ -68,14 +68,14 @@ export class CustomNodeComponent implements OnChanges {
...
+  // addFilterSubscription: Subscription;
   pluginName = '';
   isFilterNode: boolean = false;
   destroy$: Subject<boolean> = new Subject<boolean>();
   fetchedTask;
   fetchedService;
-  showPlusIcon = false;
-  showDeleteIcon = false;
+  // showPlusIcon = false;
+  // showDeleteIcon = false;
```
Comment (MonikaSharma06) [inline src/app/components/common/node-editor/custom-node/custom-node.component.ts:78]: Please remove commented code from here and other files. And there are some unused imports in this file, remove those as well.

File: src/app/components/common/node-editor/filter.ts [hunk 2]
```diff
@@ -2,24 +2,35 @@ import { ClassicPreset } from "rete";
...
+  constructor(public pseudoConnection: boolean) {
+    super();
+  }
+}
 
 export class Filter extends ClassicPreset.Node {
   height = 90;
   width = 198;
   parent?: string;
+  pseudoNode?: boolean;
```
Comment (MonikaSharma06) [inline src/app/components/common/node-editor/filter.ts:15]: Is `pseudoNode` that filter node which is withOUT connection on the canvas?

Comment (mshariq-nerd) [reply src/app/components/common/node-editor/filter.ts:15]: Yes

**Review by MonikaSharma06 — DISMISSED** [review]

**Review by hemantgarg923 — APPROVED** [review]

**Review by hemantgarg923 — DISMISSED** [review]

**Review by hemantgarg923 — APPROVED** [review]

**Review by MonikaSharma06 — APPROVED** [review]

**Pull request merged**